conan/tools/meson/meson.py

```diff
@@ -39,8 +39,10 @@ def configure(self, source_folder=None):
             cmd += ' -Dprefix="{}"'.format(self._conanfile.package_folder)
         self._run(cmd)
 
-    def build(self):
-        cmd = 'meson compile -C "{}"'.format(self._build_dir)
+    def build(self, target=None):
+        cmd = 'meson compile -C "{}" -j {}'.format(self._build_dir, cpu_count())
```

Comment:
Looks good, in the close future we will move the cpu_count to the new [conf] configuration, but this is good now, thanks.